docs/Daos.md

```diff
@@ -0,0 +1,261 @@
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT"
+});
+```
+
+And voilà, you have created a new DAO in the DAO stack on Ethereum.  The returned object `newDao` is an instance of [DAO](api/classes/DAO) and you are off and running.  But you won't run far without schemes and founders who can make proposals and vote on them, and for that, the founders will need some reputation.
+
+### Creating a new DAO with founders
+
+Create a new DAO with founders by including a "founders" array.  This will automatically mint tokens and reputation to each founder:
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  founders: [
+    {
+      // the current user
+      address: accounts[0],
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    },
+    {
+      address: founder1Address,
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    },
+    {
+      address: founder2Address,
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(40)
+    }
+  ]
+});
+```
+
+!!! note
+    It is not possible to add or remove founders.  In fact, there is no retained sense of who they even were .  They are simply addresses you supply when you create a DAO and to whom will immediately be minted tokens and reputation.  You can mint reputation or tokens to others... this would effectively be like adding them as founders.
+
+So this DAO has founders with tokens and reputation, but no way to make proposals, that is, no schemes, and thus nothing on which to vote.  The DAO needs some schemes, at minimum a [SchemeRegistrar](api/classes/SchemeRegistrarWrapper) with which you can propose to add, modify or remove other schemes.
+
+### Creating a new DAO with schemes
+
+Create a new DAO with schemes by including a "schemes" array.  The following example registers the schemes with all default parameters, permissions and voting machines.
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  schemes: [
+    { name: "SchemeRegistrar" },
+    { name: "UpgradeScheme" },
+    { name: "GlobalConstraintRegistrar" }
+  ]
+});
+```
+
+Put the last two examples together and you have a DAO with founders and schemes. Now you can _really_ go places.
+
+What if we want to configure how voting will proceed when someone submits a proposal?
+
+### Creating a new DAO overriding the default voting machine
+
+By default, `DAO.new` configures each scheme with the [AbsoluteVote](api/classes/AbsoluteVoteWrapper) voting machine and default voting parameter values.  The following example retains the default voting machine while overriding its parameters:
+
+
+```javascript
+const newDao = await DAO.new({
+  name: "My New DAO",
+  tokenName: "My new Token",
+  tokenSymbol: "MNT",
+  votingMachineParams: {
+    votePerc: 45,
```

Comment:
I have agreed with your comment since I first looked at this code.  It used to be "votePrec" (misspelled).  It at least corrected the spelling.  But this should be a code change ticket, not part in a documentation PR.

Reply:
This PR has been sitting for quite some time awaiting review while code changes continue to happen.  If I start changing code here, you're going to have to review code changes too, and I'll have to deal with increasing complications in the ongoing effort to keep this branch up-to-date with the master.  Coding the change would distract me from other higher priority coding I'm working on right now, even though it is really not an important change.

So I want a ticket.  :-)  Here it is:  https://github.com/daostack/arc.js/issues/192